test/augmentation/test_augmentation.py

```diff
@@ -4262,3 +4276,71 @@ def test_exception(self, device, dtype):
     @pytest.mark.skip(reason="not implemented yet")
     def test_module(self, device, dtype):
         pass
+
+
+class DummyMPDataset(torch.utils.data.Dataset):
+    def __init__(self):
+        super().__init__()
+        # we add all transforms that could potentially fail in
+        # multiprocessing with a spawn context below, that is all the
+        # trasnforms that define a RNG
+        self._transform = torch.nn.Sequential(
+            RandomTranslate(),
+            RandomShear(0.1),
+            RandomPosterize(),
+            RandomErasing(),
+            RandomPlanckianJitter(),
+            RandomMotionBlur(kernel_size=3, angle=(0, 360), direction=(-1, 1)),
+            RandomGaussianBlur(3, (0.1, 2.0)),
+            RandomPerspective(),
+            ColorJitter(),
+            ColorJiggle(),
+            RandomJigsaw(),
+            RandomAffine(degrees=15),
+            RandomMotionBlur3D(kernel_size=3, angle=(0, 360), direction=(-1, 1)),
+            RandomPerspective3D(),
+            RandomAffine3D(degrees=15),
+            RandomRotation3D(degrees=15),
+        )
+
+        self._resize = Resize((10, 10))
+        self._mosaic = RandomMosaic((2, 2))
+        self._crop = RandomCrop((5, 5))
+        self._crop3d = RandomCrop3D((5, 5, 5))
+        self._mixup = RandomMixUpV2()
+        self._cutmix = RandomCutMixV2()
+        self._rain = RandomRain(p=1, drop_height=(1, 2), drop_width=(1, 2), number_of_drops=(1, 1))
+
+    def __len__(self):
+        return 10
+
+    def __getitem__(self, _):
```

Comment:
i think we would like to pass device and dtype to generate the args -- but I'm not sure cc @shijianjian

Reply:
I am not sure. This is, at least in theory, completely independant of the device and dtype?